crud/delete.lua

```diff
@@ -21,14 +22,55 @@ local function delete_on_storage(space_name, key)
         return nil, DeleteError:new("Space %q doesn't exist", space_name)
     end
 
-    local tuple = space:delete(key)
-    return tuple
+    return schema.wrap_box_space_func_result(false, space, space.delete, space, key)
```

Comment:
Why "false"? Please describe for all places where you use it.

Reply:
`false`? What?

Reply:
What? Describe why did you pass "false" as the first argument here

Reply:
I think that it looks bad now (I mean passing this flag as a first argument).
I can add `schema.wrap_box_space_func_result_no_schema_hash` for such places, but it will require a comment anyway (to describe a logic "why it don't need space hash").

Reply:
Added comments to all calls.

I've also changed `schema.wrap_box_space_func_result` signature to avoid passing `space, space:<func>, space`.